account_cash_invoice/readme/DESCRIPTION.rst

```diff
@@ -1,2 +1,2 @@
-This modules allows to pay an existing Supplier Invoice / Customer Refund, or
+This modules allows to pay existing customer invoices/refunds or vendor bills/refunds, or
 to collect payment for an existing Customer Invoice from a Cash Statement.
```

Comment:
Sorry, didn't read the rest of the description. I think this line can go out now.